flavors/README.md

```diff
@@ -90,3 +209,8 @@ The naming convention is like this:
 This allows us to provide new versions of a flavor when new features become available in Exasol that are not supported in older flavors.
 
 So, in order to find the correct version of a flavor for your version of Exasol, simply take the latest container with version less than or equal to your Exasol version. So for Exasol 6.2.13, you would use the `*-EXASOL-6.2.0` flavors.
+
+## Cuda Support
+
+For the CUDA-based flavors we recommend to use the CUDA LTS Driver version 535.
```

Comment:
do we want to directly suggest 580?

Reply:
Let's continue with 535 now, as 580 is making trouble.